src/main/java/org/folio/entlinks/domain/entity/AuthoritySourceFileCode.java

```diff
@@ -0,0 +1,64 @@
+package org.folio.entlinks.domain.entity;
+
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import jakarta.persistence.UniqueConstraint;
+import java.util.Objects;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.hibernate.Hibernate;
+
+@Getter
+@Setter
+@Entity
+@AllArgsConstructor
+@NoArgsConstructor
+@Table(name = "authority_source_file_code", uniqueConstraints = {
+  @UniqueConstraint(name = "uc_authoritysourcefilecode_code", columnNames = {"code"})
+})
+public class AuthoritySourceFileCode {
+  @Id
+  @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "authority_source_file_code_id_seq")
+  @Column(name = "id", nullable = false)
+  private Integer id;
+
+  @ManyToOne(cascade = CascadeType.ALL, optional = false, fetch = FetchType.LAZY)
```

Comment:
I don't think that we should have CascadeType.ALL there, it shouldn't be possible to delete Authority